spec/support/common_actions/budgets.rb

```diff
@@ -0,0 +1,17 @@
+module Budgets
+  # spec/features/budgets/ballots_spec.rb
+  def expect_message_organizations_cannot_vote
+    #expect(page).to have_content 'Organisations are not permitted to vote.'
```

Comment:
If this expectation is not being used, I think it'd be better to remove it altogether